### ISSUE TITLE
REX-Ray 0.3.3 Volume Creation incompatible with VirtualBox 5.1

### ISSUE THREAD

**Issue by alexei-led** [codedellemc/rexray#508]
I'm using `REX-Ray` `0.3.3` version with Virtualbox (for local dev). After updating to Virtualbox 5.1, only first `rexray` VM "owns" all Virtualbox created volumes.
When trying to mount volume from second machine, I've got the following error:

```
docker: Error response from daemon: VolumeDriver.Mount: {"Error":"no device name returned"} VolumeDriver.Mount: {"Error":"no device name returned"}
```

In Virtualbox "Virtual Media Manager" UI, it's possible to see that volume indeed attached to the first machine.
Currently the only way to work is to rollback to Virtualbox 5.0.24.

**Comment by kacole2** [comment]
confirmed on v0.3.3. 0.4.3 has more servere issues where a new volume cannot even be created. steps to recreate:

**0.3.3**

1. create 2 machines using virtualbox & install REX-Ray v0.3.3

- `docker-machine create -d virtualbox rrtest01`
- `docker-machine create -d virtualbox rrtest02`
- install REX-Ray

2. on machine 1 create a new volume
- `docker volume create -d rexray --name mytest01 --opt=size=5`

3. on machine 2, try to run a container with the volume and return the error
- `docker run -ti --volume-driver=rexray -v mytest01:/mytest01 busybox`
- returned error is `docker: Error response from daemon: VolumeDriver.Mount: {"Error":"no device name returned"}.`


**0.4.3**

1. upgrade both machines to 0.4.3 and update config files if necessary 
2. restart RR
3. `rexray volume create --volumename=mytest --size=5` will return the following error:
```
ERRO[0001] error creating new volume                     host=unix:///home/docker/.libstorage/var/run/libstorage/687358817.sock inner=failed to create medium instanceID=virtualbox=af734df9-2904-4cf7-bdbe-9e3c4e760213 provider=virtualbox route=volumeCreate server=split-master-ch service=virtualbox size=5 task=0 time=1468859145593 tls=false txCR=1468859145 txID=5f08fa4d-3284-4ee0-7367-865f05190758 volumeName=mytest
ERRO[0001] error creating new volume                     host=unix:///home/docker/.libstorage/var/run/libstorage/687358817.sock inner=failed to create medium provider=virtualbox route=volumeCreate server=split-master-ch size=5 time=1468859145594 tls=false txCR=1468859145 txID=5f08fa4d-3284-4ee0-7367-865f05190758 volumeName=mytest
FATA[0001] error creating new volume
```

**Comment by kacole2** [comment]
@akutz @alexei-led I just ran a test again because I realized that I jumped the gun and my 0.4.3 configuration file was incorrect. 

I tested with version 0.4.3 on VirtualBox 5.1 and I have no issues with pre-emption or using a disk created on one host and using it on another

**Comment by kacole2** [comment]
@alexei-led have you tested with 0.4.3 yet? No issues have been indentified. We are probably not going to go back and fix older versions of code with 0.3.3. Please close if this works
